src/suparnaNJune20/ArmstrongNo.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Assignment 20 : 28th Jun 2020
+ * Program 4: WAP to check if a number is Armstrong or not.
+Example:-
+Input:- 153
+Output:-153
+hint:- (1*1*1)+(5*5*5)+(3*3*3)
+1+125+27
+153
+ */
+package suparnaNJune20;
+public class ArmstrongNo {
+	void findarmstron(int input) {
+		int amstrongNum = input, sum = 0;
+		System.out.println("Orignal  number " + input);
+		while (input != 0) {
+			sum += (input % 10) * (input % 10) * (input % 10);
+			input = input / 10;
+		}
+		if (amstrongNum == sum)
+			System.out.println(amstrongNum + " is Armstrong No");
+		else
+			System.out.println(amstrongNum + " is not Armstrong No");
+	}
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		int input = 153;
+		ArmstrongNo armstron = new ArmstrongNo();
+		armstron.findarmstron(input);
```

Comment:
We can merge line 28 and 29, try with: new ArmstrongNo().findarmstron(input);

Also, we can directly pass the hard-coded value (instead of storing it in a variable 'input') while calling the service method.

Reply:
i have taken this input variable for scanner input in next level code change.
new ArmstrongNo().findarmstron(input) - change done.

Reply:
in case of scanner class also we can write it like below.

new ArmstrongNo().findarmstron(scanner.nextInt()) , may not required separate variable.

Reply:
okay will take care of this.